libs/portal-layout-styles/src/styles/primeng/sass/theme/extensions/_calendar.scss

```diff
@@ -87,6 +108,12 @@
   }
 }
 
+.p-datepicker:not(.p-disabled) table td span:not(.p-highlight):not(.p-disabled):focus {
+  outline: 1px solid var(--primary-color);
+  outline-offset: 0;
+  box-shadow: none;
+}
```

Comment:
same thing to consider as above comments

Reply:
changed it to $calendarFocusOutline